scripts/cat/cats.py

```diff
@@ -2098,6 +2098,10 @@ def is_potential_mate(self,
         is_former_mentor = (other_cat.ID in self.former_apprentices or self.ID in other_cat.former_apprentices)
         if is_former_mentor and not game.clan.clan_settings['romantic with former mentor']:
             return False
+        
+        #current mentor
+        if other_cat.ID in self.apprentice or self.ID in other_cat.apprentice:
+            return False
```

Comment:
This same code is already present in lines 2094 - 2095.

Reply:
oh. huh. then...i'm blind sorry. to be fair i was looking at lifegen code for like 5 hours

Reply:
i wonder why that issue happened then